File: CSharp/HowMuchIsTrue.cs
namespace CSharp
{
    // Create a function which returns the number of true values there are in an array.
    // https://edabit.com/challenge/afStrS4BkCKums62B
    public static class HowMuchIsTrue
    {
        public static int CountTrue(bool[] arr)
        {
            int count = 0;

            foreach (var item in arr)
            {
                if (item == true)
                {
                    count++;
                }
            }

            return count;
        }
    }
}

File: python/how_much_is_true.py
"""
How Much is True?

Create a function which returns the number of True values in a list.

https://edabit.com/challenge/wFpi2zFGxWxfj5mZS
"""
from typing import List


def count_true(lst: List[bool]) -> int:
    return lst.count(True)
